### DISCUSSION TITLE
Orca - Bambu Studio sync

### DISCUSSION THREAD

**Discussion by LJKraus** [OrcaSlicer/OrcaSlicer#2015] [General]
I'm not sure if this is a bug, or a temporary cloud issue, or something I've done wrong.  In the past, my user filament presets for OrcaSlicer and Studio stayed in sync through the Bambu cloud.   Add/change/delete a (supported) setting or profile in one program and the other one reflected the changes.

Synching filament profiles is not working as expected today.

I’m finding that user filament preset changes in Orca 1.6.6 are available in Studio 1.7.4.52.
User presets created or edited in Studio are NOT reflected in Orca, but I can delete a user profile in Studio and it is also gone from Orca.

Logout/login, restarting both programs, removing/reinstalling both programs had no effect.

I’ve updated both Studio and OrcaSlicer in the past week, without changing any presets, so I don’t know when the sync behavior changed.

Anyone else seeing this change?  Any suggestions to restore the old behavior?

**Comment by itsKylvanas** [comment]
I am also seeing this somewhat. The filament stays the same but the OTA update applied in Studio is not in Orca?

**Comment by LJKraus** [comment]
That's how I found out that Studio presets were not syncing in the same way with Orca.  The new filament profiles were not showing in Orca, so I tried editing one in Studio with a different name, which made it a User Preset, but still did not transfer to Orca.  

Interestingly, the Bambu "configuration" update that added the filaments did NOT create new .json files in “C:\Program Files\Bambu Studio\resources\profiles\BBL\filament” where all the other "built-in" filament presets exist. So Bambu is doing _something_ different, hopefully not intentionally blocking the Orca sync.

**Reply by LJKraus** [reply]
Fixed in OrcaSlicer 1.7.0-beta.  Part of "Merged BambuStudio 1.7.4 changes to have better support for BambuLab printers."

Re-select filaments to use with the gear icon on the Filament line.
